### PR TITLE
DOC: Updated link

### LINKED ISSUE #765: Broken link in docs
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue summary

The [InneryEye WSL docs](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/docs/WSL.md) link in https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/environment.md#operating-system is broken.

### What documentation should be provided?

I guess the path should be `/docs/WSL.md`, not `/docs/docs/WSL.md`. I haven't checked other links.

[AB#6699](https://innereye.visualstudio.com/60ce1777-00d6-4015-82bc-488a0c00202f/_workitems/edit/6699)

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
Updated link in https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/environment.md#operating-system of [InneryEye WSL docs](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/docs/WSL.md)
Fixes #765

### CODE REVIEW

File: docs/environment.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 We recommend using our toolbox with [Ubuntu 20.04 LTS](https://releases.ubuntu.com/20.04/).  Most core InnerEye functionality will be stable on other operating systems, but PyTorch's full feature set is only available on Linux. All jobs in AzureML, both training and inference, run from an Ubuntu 20.04 Docker image. This means that using Ubuntu 20.04 locally allows for maximum reproducibility between your local and AzureML environments.
 
-For Windows users, Ubuntu can be set up with [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install). Please refer to the [InneryEye WSL docs](docs/WSL.md) for more detailed instructions on getting WSL set up.
+For Windows users, Ubuntu can be set up with [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install). Please refer to the [InneryEye WSL docs](/docs/WSL.md) for more detailed instructions on getting WSL set up.
```
Comment (ant0nsc) [inline docs/environment.md:7]:
```suggestion
For Windows users, Ubuntu can be set up with [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install). Please refer to the [InneryEye WSL docs](/docs/WSL.md) for more detailed instructions on getting WSL set up.
```

Comment (ant0nsc) [inline docs/environment.md:7]:
```suggestion
For Windows users, Ubuntu can be set up with [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install). Please refer to the [InneryEye WSL docs](WSL.md) for more detailed instructions on getting WSL set up.
```

**Review by ant0nsc — CHANGES_REQUESTED** [review]
Thanks for your change! I checked the issue that you referenced, it is somewhat misleading. 
I've suggested a change that fixes the underlying problem.

**Review by ant0nsc — APPROVED** [review]

**Review by peterhessey — APPROVED** [review]

**Comment by ant0nsc** [comment]
@PrajwalBorkar , I had to manually start a pipeline, please watch the outcome, and feel free to merge the result is green :-)

**Comment by peterhessey** [comment]
Made obsolete by the [RTD PR](https://github.com/microsoft/InnerEye-DeepLearning/pull/768)

**Pull request closed without merging**